docs/development_guide.md

```diff
@@ -278,6 +278,7 @@ The different chart templates are used as follows:
 - `mutatingwebhook.yaml` is used to deploy a `MutatingWebhookConfiguration` kubernetes resource that enables automatic sidecar injection
 -  `grafana-*.yaml` chart templates are used to deploy a Grafana instance when grafana installation is enabled
 - `prometheus-*.yaml` chart templates are used to deploy a Prometheus instance when prometheus installation is enabled
+- `fluentbit-configmap.yaml` is used to deploy a fluent bit sidecar container with the OSM controller when fluent bit is enabled
```

Comment:
Isn't the fluent bit configmap used to provide various configurations to fluent bit?